src/components/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className="footer-div" id="github-link-container">
+        <a
+          href="https://github.com/AccessibleForAll/ColorPaletteCombos"
+          className="github-link"
+        >
+          GitHub
+        </a>
+        <svg
```

Comment:
I have imported react icons package into the project. Please swap this out for the `<AiOutlineGithub />` component instead. It won't need an aria label as it's purely decorative.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
replaced ✔